modulus/models/dlwp_healpix_layers/healpix_blocks.py

```diff
@@ -493,6 +493,54 @@ def forward(self, x):
         # return second convnext result
         return self.skip_module2(x1) + self.convblock2(x1)
 
+class Multi_SymmetricConvNeXtBlock(th.nn.Module):
+    """
+    Support "n_layers" in SymmetricConvNeXtBlock.
+    Create a multi-layer SymmetricConvNeXtBlock with 'n_layers' specified in the config files
+    Introduce a new class to prevent conflicts with existing trained models.
```

Comment:
Can you change the doc string here to something that is more descriptive/ won't become out-dated? The comments here seem more relevant to a PR than in code.

 e.g. "Class for creating multi-block SymmetricConvNeXtBlock. Defaults to all SymmetricConvNeXtBlocks having same parameters".

Reply:
Thanks Zac. The comment has been updated.